examples/own_dataset/train.py

```diff
@@ -50,6 +54,26 @@ def __call__(self, atoms, adjs):
         return x
 
 
+class ScaledAbsError(object):
+
+    def __init__(self, scale='standardize', ss=None):
+        self.scale = scale
```

Comment:
Please raise `ValueError` when `scale` is neither `standardize` nor `none`.

Reply:
I removed scale option, only check scaler